dbtc/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.11.6"
+__version__ = "0.12.0"
```

Comment:
This can prolly just be 0.11.7 as there are no breaking changes (unless I missed something).  Other than that, looks good (and so sorry that it's taken so long for me to review).